### ISSUE TITLE
Save as Templates for later use

### ISSUE THREAD

**Issue by HStep20** [meeb/tubesync#172]
An Enhancement Request, rather than an issue

It would be nice if there were some kind of 'template' setup when you go to create a new source. I download Channels and Playlists, as most people likely do, but I format the title different for each one. It would be a nice UX feature if you could create a template that would fill out all of the 'new source' fields based on however it was set up in the template. 

For example, my channel template may look like this, leaving me to fill out the channel name and directory when a new channel is added:
```
Type | YouTube channel
Name | 
Key | simplysarc
Directory | {channel_name}
Media format | {yyyymmdd} - {title_full}.{ext}
Index schedule | Every 24 hours
Download media? |  Yes
Source resolution | 2160p (4K)
Source video codec | VP9
Source audio codec | OPUS
Prefer 60FPS? |  Yes
Prefer HDR? |  Yes
Output extension | mkv
Fallback | Get next best resolution but at least HD
Copy thumbnails? |  Yes
Write NFO? |  Yes
Delete old media | No, keep forever
```

Same for playlists, where I may use a different Media Format and Directory structure, like {channel_name}/{playlist_title}

**Comment by meeb** [comment]
Yep sounds reasonable, I think this may also be a dupe but I'll add it to the wishlist for now. Cheers.
